src/groups/bmq/bmqp/bmqp_protocol.t.cpp

```diff
@@ -1262,9 +1262,9 @@ static void test3_flagUtils()
                 bmqp::StorageHeaderFlags::Enum currFlag =
                     static_cast<bmqp::StorageHeaderFlags::Enum>(currFlagVal);
 
-                PVV(test.d_line
-                    << ": Testing: " << "    StorageHeaderFlagUtil::isSet('"
-                    << flagsString << "', " << currFlag << ")");
+                PVV(test.d_line << ": Testing: "
```

Comment:
Sometimes running 'clang-format -i file_name.cpp' will result in extraneous changes that don't match the formatting checks. Is there a better way to address sometimes occasional changes other than running it through the CI and then going back to fix it?